spring-rabbit/src/test/java/org/springframework/amqp/rabbit/connection/SSLConnectionTests.java

```diff
@@ -85,9 +93,34 @@ public void testNoAlgNoProps() throws Exception {
 		fb.setUseSSL(true);
 		fb.afterPropertiesSet();
 		fb.getObject();
-		verify(rabbitCf).useSslProtocol();
+		verify(rabbitCf).useSslProtocol(Mockito.any(SSLContext.class));
+	}
+
+	@Test
+	public void testuseSslProtocolShouldNotbecalled() throws Exception {
```

Comment:
Minor, but would be better to follow with camel case formatting for naming.
I mean this is preferable: `testUseSslProtocolShouldNotBeCalled`